src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypePlainAndArrays.java

```diff
@@ -0,0 +1,25 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
+
+public class InputIllegalTypePlainAndArrays {
```

Comment:
please define configuration in Input file (in all new Input files).
To let anybody open input file , read config, and see how it should work.

example (see others input files):
`//configuration: "illegalClassNames": List, GregorianCalendar, java.io.File, SubCalendar, ArrayList`

Reply:
done